spec/acceptance/README.md

```diff
@@ -27,9 +27,14 @@ SPEC_OPTS='--tag acceptance' SPEC_HELPER_LOAD_METASPLOIT=false bundle exec rspec
 ```
 
 Run a specific Meterpreter/module test Unix / Windows:
+
+# Bash command:
```

Comment:
The headings here isn't hierarchically correctly 👀 

I think either more `#` needs added to fix the header depth, or maybe it shouldn't be a heading as the other examples aren't 🤔